features/aanschrijfwijze.feature

```diff
@@ -2,17 +2,17 @@
 
 # User story #13
 Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze van mijn burgers
-  Attribuut aanschrijfwijze bij een persoon wordt gevuld door de provider om op deze wijze op eenduidige wijze een persoon te kunnen aanschrijven.
-  Bij het samenstellen van de aanschrijfwijze worden academische titels vooralsnog niet opgenomen. Academische titels zijn geen authentiek gegeven en daarom buiten scope geplaatst.
+  De attributen aanschrijfwijze.naam en aanschrijfwijze.aanspreekvorm bij een persoon worden gevuld door de provider om op deze wijze op eenduidige wijze een persoon te kunnen aanschrijven.
+  Bij het samenstellen van de aanschrijfwijze.naam worden academische titels vooralsnog niet opgenomen. Academische titels zijn geen authentiek gegeven en daarom buiten scope geplaatst.
```

Comment:
Hier is de toevoeging .naam niet nodig denk ik, juist fout, want ook een academische titel kan een aanspreekvorm hebben (De hooggeleerde heer/vrouwe, De weledelzeergeleerde heer/vrouwe)

Reply:
Ah ja, ik begrijp je.

Reply:
Aangepast